maestro-client/src/main/java/maestro/Maestro.kt

```diff
@@ -181,7 +170,7 @@ class Maestro(private val driver: Driver,
     ) {
         LOGGER.info("Tapping on element: $element")
 
-        val hierarchyBeforeTap = waitForAppToSettle(initialHierarchy)
+        val hierarchyBeforeTap = waitForAppToSettle(initialHierarchy) ?: initialHierarchy
```

Comment:
We already pass `initialHierarchy` to `waitForAppToSettle`, why do we need an `?:` operator here?

Reply:
for iOS driver might return null right now in case of screenshot comparison and function parameter is also optional